src/main/java/com/example/home_work_8/geometry/Circle.java

```diff
@@ -0,0 +1,40 @@
+package com.example.home_work_8.geometry;
+
+public class Circle implements Geometry {
+
+    final static double pi = Math.PI;
+    private int circleS;
+    private int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
+        this.circleS = figuraArea(radius, 0);
```

Comment:
If you need to initialize this field, do it through the constructor as you did with `radius`